### PR TITLE
MAIN-5540 Log changes for Junior helper group

### PR DESCRIPTION
**Pull request by armonr**
And add username to log

/CC @Grunny @garthwebb @jsutterfield

### CODE REVIEW

**Comment by armonr** [comment]
Reverted my change forbidding password to be updated for admin/util through prefs.

File: extensions/wikia/UserLogin/UserLoginSpecialController.class.php [hunk 1]
```diff
@@ -615,8 +615,10 @@ public function mailPassword() {
 			return;
 		}
 
-		if ( !empty( array_intersect( F::app()->wg->AccountAdminGroups, $user->getGroups() ) ) ) {
-			\Wikia\Logger\WikiaLogger::instance()->warning("Junior helper cannot change account info");
+		if ( !empty( array_intersect( F::app()->wg->AccountAdminGroups, $user->getAllGroups() ) ) ) {
```
Comment (garthwebb) [inline extensions/wikia/UserLogin/UserLoginSpecialController.class.php]: Why was this changed to `getAllGroups`?  The `getAllGroups` method returns _all_ groups defined in the system, not just the groups for that user.
